tests/Entity/AdherentTest.php

```diff
@@ -116,7 +116,7 @@ public function testUserWithLegislativeCandidateRole()
         $this->assertFalse(in_array('ROLE_LEGISLATIVE_CANDIDATE', $adherent->getRoles()));
 
         $adherent = $this->createAdherent('john.smith@en-marche.fr');
-        $this->assertTrue(in_array('ROLE_LEGISLATIVE_CANDIDATE', $adherent->getRoles()));
+        $this->assertFalse(in_array('ROLE_LEGISLATIVE_CANDIDATE', $adherent->getRoles()));
```

Comment:
`$this->assertContains('ROLE_LEGISLATIVE_CANDIDATE', $adherent->getRoles());`

Reply:
@Remg I want to test the opposite, but 👍.
Fixed, thanks!